styles.css

```diff
@@ -0,0 +1,101 @@
+@import url(./css/utilities/variables.css);
+@import url(./css/layouts/navbar.css);
+@import url(./css/layouts/footer.css);
+html{
+    background-color: var(--lightest-green);
+    font-family: var(--acme-font);
+}
+
+/* Header */
+.hero-header{
+    width: 100%;
+    height: 80vh;
+    color: var(--secondary);
+    background-image: url(./assets/Images/farm.jpg);
+    background-repeat:no-repeat;
+    background-size:cover;
+    background-position: center;
+    position: relative;
+}
+
+.banner-title{
```

Comment:
just a suggestion that if that banner title and sub-title is in the middle it would look really good.